dspace-oai/src/main/java/org/dspace/xoai/app/XOAI.java

```diff
@@ -719,4 +731,45 @@ private static void usage() {
         }
     }
 
+    private boolean isTest() {
```

Comment:
Is this isTest used often in the DSpace?

Reply:
It is from version5, we use isTest but with the different implementation. Should I removed it?

Reply:
If this is not used I think we should remove it